gwy/templates/main.go.tmpl

```diff
@@ -14,7 +14,7 @@ import (
 	"time"
 
 	"github.com/gorilla/handlers"
-	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
```

Comment:
is it a breaking change?

Reply:
The gateway generation is actually currently broken without this change with the compilation of the generated gateway failing with 

```
cannot use gwmux (type *"github.com/grpc-ecosystem/grpc-gateway/runtime".ServeMux) as type *"github.com/grpc-ecosystem/grpc-gateway/v2/runtime".ServeMux in argument to namely_reportbuilder_v1.RegisterReportsGrpcServiceHandlerFromEndpoint
```

This is because grpc-gateway master is currently v2 and the module name adds a v2 to the import path.

Reply:
Good point, I should have clarified. It's actually broken right now because the package name changed. we confirmed the image works with this change.